README.md

```diff
@@ -294,7 +294,7 @@ CDC including this GitHub page may be subject to applicable federal law, includi
 
 ### Database
 
-For database documentation [go here](/docs/database.md)
+For database documentation `/docs/database.md`
```

Comment:
I broke this hyperlink because it wasn't working and said 'the document does not exist' when it does.

Reply:
The original link works for me

Reply:
Please feel free to change it if most others can use the link